packages/components/pie-button/src/button.scss

```diff
@@ -1,3 +1,60 @@
-@use '@justeat/fozzie/src/scss/fozzie' as f;
+@import '@justeat/pie-design-tokens/dist/jet.scss';
 
-@include f.buttons();
+.o-btn {
+    border-radius: $radius-rounded-e;
```

Comment:
do we want to namespace the variables like we have else where i.e. `v.$radius-rounded-e` or similar?